i18n/chs/package.i18n.json

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I believe if the i18n files are not present, vscode-nls will use English strings.  You could omit them for now.  The OneLocBuild pipeline should result in a PR being generated that adds them.